radixdlt-java/src/main/java/com/radixdlt/client/core/network/RadixJsonRpcClient.java

```diff
@@ -77,19 +72,23 @@ public RadixJsonRpcClient(WebSocketClient wsClient) {
 			.publish()
 			.refCount();
 
-		if (!CHECK_API_VERSION) {
-			this.serverApiVersion = Single.just(API_VERSION);
-		} else {
-			this.serverApiVersion = jsonRpcCall("Api.getVersion")
-				.map(result -> result.getAsJsonObject().get("version").getAsInt())
-				.cache();
-		}
+		this.serverApiVersion = SingleSubject.create();
+		this.universeConfig = SingleSubject.create();
+	}
 
+	void fetchClientInformation() {
```

Comment:
This function is still calling subscribe on it's own when really that should be done with a returned Observable or Single or Completable. Otherwise you lose the ability to dispose or catch errors.

Reply:
Ignore all that, has been removed.